toolchain/base/index_base.h

```diff
@@ -56,17 +56,8 @@ struct IdBase : public AnyIdBase, public Printable<IdT> {
   // NOLINTNEXTLINE(readability-identifier-naming)
   static const IdT& None;
 
-  auto Print(llvm::raw_ostream& out) const -> void {
-    out << IdT::Label;
-    if (has_value()) {
-      out << index;
-    } else {
-      out << "<none>";
-    }
-  }
-
   // TODO: Make Print() do the hex thing for all IDs and remove this function.
```

Comment:
Remove TODO?

Reply:
Done in 20e162054093ace27c6350e6ab0b66a8499eeec4